holidays/ical.py

```diff
@@ -0,0 +1,140 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+import uuid
+from datetime import datetime, timezone
+
+
+class ICalExporter:
+    def __init__(self, holidays_object, return_bytes=False, ical_timestamp=None):
```

Comment:
I think it's better to move the `return_bytes` parameter to `generate()` method. Or create separate methods like `as_string()` and `as_bytes()`.

Reply:
Fixed.